commit 5c933890126f3597335930681260acab604c0b7e (refs/changes/13/3313/2)
Author: Dominik Hassler <hadfl@omniosce.org>
Date:   2018-02-08T13:16:14-05:00 (1 year, 8 months ago)
    
    9027 Makefiles need to specify C99 mode consistently (missed lx_lockd)
    Reviewed by: Dan McDonald <danmcd@joyent.com>
    Reviewed by: Jason King <jbk@joyent.com>
    Approved by: Dan McDonald <danmcd@joyent.com>

diff --git a/usr/src/lib/brand/lx/lx_lockd/Makefile b/usr/src/lib/brand/lx/lx_lockd/Makefile
index e535a30877..debefdf815 100644
--- a/usr/src/lib/brand/lx/lx_lockd/Makefile
+++ b/usr/src/lib/brand/lx/lx_lockd/Makefile
@@ -30,7 +30,7 @@ ROOTBIN =	$(ROOTBRANDDIR)
 CLOBBERFILES =	$(OBJS) $(ROOTPROG)
 
 CPPFLAGS +=	-I$(SRC)/cmd/fs.d/nfs/lib
-C99MODE =	$(C99_ENABLE)
+CSTD =		$(CSTD_GNU99)
 LDLIBS +=	-lnsl -lsocket
 
 CERRWARN += -_gcc=-Wno-parentheses
